inc/cloner/class-cloner.php

```diff
@@ -1343,6 +1343,10 @@ protected function cloneMetadata() {
 				}
 			} elseif ( $key === 'pb_title' ) {
 				update_post_meta( $metadata_post_id, $key, $this->targetBookTitle );
+			} elseif ( $key === 'pb_institutions' ) {
+				foreach ( $value as $institution ) {
+					add_post_meta( $metadata_post_id, $key, $institution );
+				}
```

Comment:
Dalcin, here a proposal. In order to avoid:
- repetition of https://github.com/pressbooks/pressbooks/pull/2587/files#diff-6917f7de9f5467b4f4fb047e37161b8a207f4430ddffaaf6086f8698d196f469R1341-R1343
- an extra ifelse non-tested

we could add the `pb_institutions` to `$metadata_array_values` in https://github.com/pressbooks/pressbooks/pull/2587/files#diff-6917f7de9f5467b4f4fb047e37161b8a207f4430ddffaaf6086f8698d196f469R1325, and
here: https://github.com/pressbooks/pressbooks/pull/2587/files#diff-6917f7de9f5467b4f4fb047e37161b8a207f4430ddffaaf6086f8698d196f469R1341-R1343
 we could add the extra condition like this:
```php
$values = $key !== 'pb_institutions' ? explode( ', ', $value ) : $value;
foreach ( $values as $v ) {
	add_post_meta( $metadata_post_id, $key, $v );
}
```

Reply:
For now I think it makes sense to do this way. If we add extra information to the institution we'll likely need to change this.